public/index.html

```diff
@@ -54,5 +42,18 @@
     <noscript>You need to enable JavaScript to run this app.</noscript>
     <div id="root"></div>
     <div id="portal-root"></div>
+    <!-- GTAG -->
+    <script defer>
+      ;(function (w, d, s, l, i) {
+        w[l] = w[l] || []
+        w[l].push({ 'gtm.start': new Date().getTime(), event: 'gtm.js' })
+        var f = d.getElementsByTagName(s)[0],
+          j = d.createElement(s),
+          dl = l != 'dataLayer' ? '&l=' + l : ''
+        j.defer = true
```

Comment:
`defer` won't work for inline script, beside we already have defer in line 53.

Reply:
my fault...

Reply:
my fault...